tests/normal.t.pl

```diff
@@ -28,10 +28,10 @@ sub runTests
     my $hardlinks = 1;
     if($mode eq 'standard')
     {
-        &mount("--standard");
+        &mount("--standard -o direct_io");
     } elsif($mode eq 'paranoia')
     {
-        &mount("--paranoia");
+        &mount("--paranoia -o direct_io");
```

Comment:
How do network file systems on FreeBSD handle this, sshfs for example? There, you cannot assume that the file does not change behind your back.